package.json

```diff
@@ -75,6 +75,7 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "randomcolor": "^0.5.3",
+    "regenerator-runtime": "^0.11.0",
```

Comment:
Move this to both dev deps, and peer deps instead of here.

Reply:
done